### PR TITLE
Cilium 1.13.6

### PR DESCRIPTION
**Pull request by njuettner**
Related Issue: https://github.com/giantswarm/flexshopper/issues/397

This PR:

- applies the changes for the latest patch version for Cilium 1.13

### Testing

Description on how cilium can be tested.

- [x] fresh install works
  - [x] AWS
  - [ ] Azure
  - [ ] KVM
- [x] upgrade from previous version works
  - [x] AWS
  - [ ] Azure
  - [ ] KVM

#### Other testing

Description of features to additionally test for cilium installations.

- [ ] check reconciliation of existing resources after upgrading
- [ ] X still works after upgrade
- [ ] Y is installed correctly

### Checklist

- [ ] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

**Review by whites11 — APPROVED** [review]

**Comment by fiunchinho** [comment]
Wow that's a lot of changes. I'm fine merging if it's been tested on CAPI clusters too.

**Comment by njuettner** [comment]
I just ran a test again with Cilium 1.13.6 for CAPA, tests are passing https://github.com/giantswarm/cluster-aws/pull/369.

File: helm/cilium/values.yaml [hunk 1]
```diff
@@ -1603,12 +1669,13 @@ prometheus:
...
+    # -- Set to `true` and helm will not check for monitoring.coreos.com/v1 CRDs before deploying
+    trustCRDsExist: false
   # -- Metrics that should be enabled or disabled from the default metric
   # list. (+metric_foo to enable metric_foo , -metric_bar to disable
   # metric_bar).
-  # ref: https://docs.cilium.io/en/stable/operations/metrics/#exported-metrics
-  metrics: 
-   - +cilium_bpf_map_pressure
+  # ref: https://docs.cilium.io/en/stable/observability/metrics/
+  metrics: ~
```
Comment (AndiDog) [inline helm/cilium/values.yaml:1678]: @njuettner @paurosello could this be a mistake? We specifically introduced this in https://github.com/giantswarm/cilium-app/commit/a321566c834a1efa211e2bcabf5a58a1c4e5b442.

Comment (paurosello) [reply helm/cilium/values.yaml:1678]: yes, we need that back in

**Pull request merged**